test/diffbinder.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <fstream>
+#include <stdio.h>
+#include <vector>
+#include <string.h>
+
+int skip(const std::string& s1)
```

Comment:
Here and below: could you please use the same `const` notation as rest of Binder code? (if you curious, please note that there is a strong technical reason for preferring this notation, i think this comment explains well https://stackoverflow.com/a/5503602/7243597)

Reply:
Done.

Reply:
👍